app/controllers/katello/api/v2/api_controller.rb

```diff
@@ -44,8 +44,12 @@ def scoped_search(query, default_sort_by, default_sort_order, options = {})
       total = query.count
       query = resource.search_for(*search_options).where("#{resource.table_name}.id" => query)
 
-      query = query.select(group).group(group) if group
-      sub_total = query.count
+      if group.present?
+        query = query.select(group).group(group)
```

Comment:
just curious, is this change to check for blank arrays/relations as opposed to checking just for nil?